DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json

```diff
@@ -100,6 +100,7 @@
         },
         "demos": [
             "fTSWithStaticIndexSingleField",
+            "fTSQuerySearchOperators",
```

Comment:
place before the static indexes demos - keep order as in C#

Reply:
Done

Reply:
Pls keep the same order as in the C# demos.json file
The file is located in `Controlers\Demos`